src/R/Components/Test/InteractiveWorkflow/RInteractiveWorkflowCommandTest.cs

```diff
@@ -106,6 +106,7 @@ public class RInteractiveWorkflowCommandTest : IDisposable {
             command.Should().BeInvisibleAndDisabled();
 
             using (await UIThreadHelper.Instance.Invoke(() => _workflow.GetOrCreateVisualComponentAsync())) {
+                using (await _workflow.RSession.BeginInteractionAsync()) { }
```

Comment:
I've seen this issue too.
Not part of this change, but maybe we should set `RSession._initializationTcs` not in `IRCallbacks.Connected`, but in `IRCallbacks.ReadConsole`?